inventory/sample.osp.example.com.d/inventory/group_vars/all.yml

```diff
@@ -93,18 +93,7 @@ rhsm_password: ''
 # leave commented out if you want to `--auto-attach` a pool
 #rhsm_pool: ''
 
-# NOTE: these need to be declared in this file because openshift_hosted_registry_storage_kind
-# is needed playhosts: localhost; OSEv3. The remainder are here for grouping purposes
-#
-openshift_hosted_registry_storage_kind: openstack
-openshift_hosted_registry_storage_access_modes: ['ReadWriteOnce']
-openshift_hosted_registry_storage_openstack_filesystem: ext4
-#openshift_hosted_registry_storage_openstack_volumeID: automatically set from osp_volumes creation
-#openshift_hosted_registry_storage_volume_size: automatically set from {{osp_volumes.size}}
-
-osp_volumes:
-- name: "registry-vol"
-  display_description: "Registry Volume"
-  display_name: "registry-vol"
-  size: 50
-  purpose: "ose_registry"
+# enable automatic provisioning of cinder registry volume on OSP
+casl_storage_registry_kind: 'cinder'
+casl_storage_registry_create: True
+#casl_registry_size: 20
```

Comment:
you can use the `openshift_hosted_registry_storage_volume_size` inventory variable instead of making a `casl` specific one